frontend/pages/blog/index.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <main class="py-6 container">
+    <section class="text-center">
+      <h1>Blog</h1>
+      <p class="pt-2">
+        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+        eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
+        voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
+      </p>
+    </section>
+
+    <section class="pt-5 row">
+      <div class="col-lg-3 pb-5 pb-lg-0">
+        <h3>Filter</h3>
+      </div>
+
+      <b-overlay
+        :show="isLoading"
+        rounded
+        opacity="0.6"
+        spinner-small
+        spinner-variant="primary"
+        class="col-lg-9 mx-3 mx-md-0"
+      >
+        <template v-if="!isLoading">
+          <p v-if="!blogPosts.length" class="text-muted">
+            Wir haben keine Blog Beiträge gefunden.
+          </p>
+
+          <div class="row">
+            <b-card-group columns>
+              <BlogPost
+                v-for="post of blogPosts"
+                :key="post.id"
+                class="mb-5"
+                :title="post.title"
+                :excerpt="
+                  post.description.length > excerptLength
+                    ? `${post.description.slice(0, excerptLength)}...`
+                    : post.description
+                "
+                :src="
+                  post.featuredImage
+                    ? getStrapiMediaUrl(post.featuredImage.url)
+                    : ''
+                "
+                :post-id="post.id"
+              />
+            </b-card-group>
+          </div>
+        </template>
+      </b-overlay>
+    </section>
+  </main>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from '@nuxtjs/composition-api'
+import BlogPost from '~/components/BlogPost.vue'
+import { IBlogPost } from '~/types/blog'
+import { getStrapiMediaUrl, useAxios } from '~/utils/strapi'
+
+export default defineComponent({
+  components: { BlogPost },
+  layout: 'blog',
+  setup() {
+    const isLoading = ref(true)
+    const blogPosts = ref<IBlogPost[]>([])
+
+    const { axios } = useAxios()
+
+    axios
+      .get<IBlogPost[]>('blog-posts')
+      .then(({ data }) => {
+        blogPosts.value = data
+      })
+      .catch((e) => {
+        console.error(e)
```

Comment:
Why will errors only be printed to console? Does the user get no feedback, if there is an error?